modules/accounts/email.nix

```diff
@@ -133,6 +133,21 @@ let
           Configuration for secure connections.
         '';
       };
+
+      authentication = mkOption {
+        type = types.enum [
+          "password"
+          "encrypted-password"
```

Comment:
What does `encrypted-password` mean?

Reply:
I only know it as one of the methods suggested in the Thunderbird UI, but I'd guess it means sending an encrypted password over a plain (non-TLS/STARTTLS) session